util/start_test

```diff
@@ -644,7 +645,7 @@ endif
 if ($?CHPL_TEST_UTIL_DIR) then
     set utildir = "$CHPL_TEST_UTIL_DIR"
 else
-    set utildir = "$CHPL_HOME/util"
+    set utildir = `dirname $0`
```

Comment:
Actually, if start_test used the old chplenv scripts.. they wouldn't be compatible since we switched them to python. Hmm, I really have no idea how release over release manages to work. 

I guess this is ok?